pytest/requirements.txt

```diff
@@ -1,5 +1,6 @@
 PyGithub
 base58
+cython
```

Comment:
Are we actually using this for anything? I couldn’t find any usage.

Reply:
So scikit-learn requires cypthon. Very nice of them to include that in their dependencies.